test/packetimpact/testbench/connections.go

```diff
@@ -113,17 +113,17 @@ func NewTCPIPv4(t *testing.T, dut DUT, outgoingTCP, incomingTCP TCP) TCPIPv4 {
 		WindowSize: Uint16(32768),
 		SrcPort:    &localPort,
 	}
-	if err := newOutgoingTCP.merge(outgoingTCP); err != nil {
+	if err := newOutgoingTCP.MERGE(outgoingTCP); err != nil {
```

Comment:
How come MERGE is in all caps?  I don't see a change in the definition of this function so does it even compile?